tools/c7n_azure/c7n_azure/query.py

```diff
@@ -105,15 +109,21 @@ def source_type(self):
 
     def resources(self, query=None):
         key = self.get_cache_key(query)
-        resources = self.augment(self.source.get_resources(query))
+        resources = self.source.get_resources(query)
```

Comment:
What was wrong with the augment call? I think it defaults to a no-op unless someone implements it for their resource.

Reply:
The problem was augment wasn't a method in this class, it just happened to be implemented in child classes so an error was never thrown. I can add a no op augment onto this class that does nothing though! I couldn't decide which solution I liked better.

Reply:
Slept on it and changed my mind. Putting no op augment on this class.

Reply:
Yep that is perfect